src/table/src/TableRow.js

```diff
@@ -82,22 +92,24 @@ export default class TableRow extends PureComponent {
     } = this.props
 
     return (
-      <Pane
-        display="flex"
-        {...(isSelectable
-          ? {
-              'aria-selected': isHighlighted,
-              'aria-current': isSelected,
-              css: { ...selectableRowStyle, ...css },
-              tabIndex: 0
-            }
-          : { css })}
-        onClick={this.handleClick}
-        onKeyPress={this.handleKeyPress}
-        {...props}
-      >
-        {children}
-      </Pane>
+      <div ref={this.handleRef}>
```

Comment:
I'm using a wrapping `div` here so that we can attach a ref to the underlying node and avoid a call to `findDOMNode` which is discouraged for React 16 onward.

Reply:
Not sure why we would need to do to this? If you need the ref of the Pane you can use `innerRef={ref => this.paneRef = ref}`. Eslint will probably complain, so you might have to create a func to do it `innerRef={this.onRef}`

Reply:
Thanks @jeroenransijn. I took this approach because using a wrapping `div` was seemingly the easiest way preserve encapsulation. However, as you pointed out, `Pane` (via `ui-box`) has already been wired up to expose a `ref` prop that we can use. I wasn't aware of this, and this approach works just fine as well.

For reference:
https://github.com/segmentio/ui-box/blob/master/src/box.js#L50-L54

I'll amend my commit to use this approach, and include a new storybook example that will make it easier to verify the bug has been fixed.